src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X500NameEncoder.cs

```diff
@@ -178,7 +177,7 @@ private enum ParseState
 
         private static List<byte[]> ParseDistinguishedName(
             string stringForm,
-            List<char> dnSeparators,
+            string dnSeparators,
```

Comment:
The only thing `dnSeparators` is used for is for calling `.Contains(char)` which is already ordinal based, so nothing else needed to change for changing to a `string`.